nano/templates/pda.tmpl

```diff
@@ -348,7 +348,8 @@ Used In File(s): \code\game\objects\items\devices\PDA\PDA.dm
             </div>
             <div class="itemContent">
                 {{:helper.link('Clear', 'trash', {'cartmenu' : "1", 'choice' : "Status", 'statdisp' : "blank"}, null, null)}}
-                {{:helper.link('Shuttle ETA', 'gear', {'cartmenu' : "1", 'choice' : "Status",'statdisp' : "shuttle"}, null, null)}}
+                <!-- VOREStation Edit TFF 21/12/19 - Virgo = Tram. Not shuttle -->
+		{{:helper.link('Tram ETA', 'gear', {'cartmenu' : "1", 'choice' : "Status",'statdisp' : "shuttle"}, null, null)}}
```

Comment:
Indentation is inconsistent which bugs the hell out of me.

Reply:
I noticed that and double checked it. It looks like it still has the correct number of tabs/indents/whatever when I opened it up in a text editor. It also works in testing without any trouble.

Reply:
It seems like lines above it use spaces, while yours uses tabulation

Reply:
https://cdn.discordapp.com/attachments/528973352728264714/660945991591002132/Capture_2019-12-29-15-41-38.png

Reply:
That might be difference in text viewers. I literally tried to select the sections of the identation here in git and it showed that one line was filled with spaces and one below was tabulations.

Reply:
All I did was change the shuttle word in the tmpl to tram. And add a Vorestation edit comment

Reply:
Also, the screenshot was made from web editor view.

Reply:
That should do it, I think.